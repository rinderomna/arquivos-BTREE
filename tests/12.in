7 tipo2 binario12.bin indice12.bin 1
851 2020 18 "PR" "SANTA TEREZINHA DO ITAIPU" "FIAT" "STRADA FREEDOM 13CS"
