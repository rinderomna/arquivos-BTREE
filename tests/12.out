19379.670000
1382.330000
